aten/src/ATen/native/Linear.cpp

```diff
@@ -16,6 +16,10 @@
 
 namespace at { namespace native {
 
+Tensor bias(const Tensor& input, const Tensor& bias) {
+  return input.add(bias);
```

Comment:
Note: the `+` operator is overloaded for tensors, so `input + bias` works here as well.